assets/admin/students/student-modal/index.js

```diff
@@ -11,6 +11,48 @@ import { __ } from '@wordpress/i18n';
 import CourseList from './course-list';
 import InputControl from '../../../blocks/editor-components/input-control';
 
+const AddButton = (
+	<Button className="sensei-student-modal__action--add" variant="primary">
+		{ __( 'Add to Course', 'sensei-lms' ) }
+	</Button>
+);
+
+const RemoveButton = (
+	<Button className="sensei-student-modal__action--remove">
+		{ __( 'Remove from Course', 'sensei-lms' ) }
+	</Button>
+);
+
+const ResetButton = (
+	<Button className="sensei-student-modal__action--remove">
+		{ __( 'Reset or Remove the student(s) progress', 'sensei-lms' ) }
+	</Button>
+);
+
+const POSSIBLE_ACTIONS = {
+	add: {
+		description: __(
+			'Select the course(s) you would like to add students to:',
+			'sensei-lms'
+		),
+		button: AddButton,
+	},
+	remove: {
+		description: __(
+			'Select the course(s) you would like to remove students from:',
+			'sensei-lms'
+		),
+		button: RemoveButton,
+	},
+	'reset-progress': {
+		description: __(
+			'Select the course(s) you would like to reset the students progress:',
```

Comment:
This should be "Select the course(s) you would like to reset or remove progress for:"